From 8741946870a8894e495bb5b9a91b0ccfa75afc9c Mon Sep 17 00:00:00 2001
From: Simon <webmaster@zutroll.de>
Date: Fri, 22 Feb 2019 14:53:02 +0100
Subject: [PATCH] Sqlite https url change

---
 rules/sqlite.make | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rules/sqlite.make b/rules/sqlite.make
index 97f32e0ce..689e2cf66 100644
--- a/rules/sqlite.make
+++ b/rules/sqlite.make
@@ -25,7 +25,7 @@ SQLITE_VERSION	:= 3270100
 SQLITE_MD5	:= cb72c5f93235cd56b18ee2aa1504cdaf
 SQLITE		:= sqlite-autoconf-$(SQLITE_VERSION)
 SQLITE_SUFFIX	:= tar.gz
-SQLITE_URL	:= http://www.sqlite.org/2019/$(SQLITE).$(SQLITE_SUFFIX)
+SQLITE_URL	:= https://www.sqlite.org/2019/$(SQLITE).$(SQLITE_SUFFIX)
 SQLITE_SOURCE	:= $(SRCDIR)/$(SQLITE).$(SQLITE_SUFFIX)
 SQLITE_DIR	:= $(BUILDDIR)/$(SQLITE)
 SQLITE_LICENSE	:= public_domain
-- 
2.14.1.windows.1

